### ISSUE TITLE
Problem with text rendering in high resolution?

### ISSUE THREAD

**Issue by drewish** [cwhitney/PretzelGui#22]
I noticed that one of the samples have `settings->setHighDensityDisplayEnabled();` but one app has it commented out.

Disabled:
<img width="1136" alt="screen shot 2017-01-10 at 11 30 15 pm" src="https://cloud.githubusercontent.com/assets/68750/21838009/d2cd3d30-d78c-11e6-8d8f-3361375b296c.png">

Enabled:
<img width="1136" alt="screen shot 2017-01-10 at 11 30 34 pm" src="https://cloud.githubusercontent.com/assets/68750/21838008/d2b97214-d78c-11e6-87bf-580b7f68e3b0.png">

Text seems equally blurry in both cases but in the enabled case it seems more desaturated.

Curious if you've any idea what's going on with it.

**Comment by cwhitney** [comment]
Will take a look. Pretzel is due for some sprucing up overall, this is a good place to start

**Comment by drewish** [comment]
I wonder if it's just a a problem with TextureFont. i started testing out TextureFontApp and it seems to be similarly afflicted at smaller sizes. I posted some screenshots on: http://discourse.libcinder.org/t/pixalated-font-on-retina-display/435/5

Something like https://github.com/chaoticbob/Cinder-SdfText might give better results but I'm not sure it'd make sense to have this block depending that.

**Comment by cwhitney** [comment]
A hacky solution might be to draw the type twice as large and draw it at 50% if that's the case.

**Comment by drewish** [comment]
yeah of that'd fix it, it seems like something the TextureFont should be doing. i'll try to do some experimentation since i've got another feature i'm working on that needs to do some text rendering.

**Comment by lmaw** [comment]
Hi,
Change these lines in renderTextInternal(PretzelGlobal.cpp)

`
guiFont->drawString(text, ci::vec2(-textSize.x, 0) + pos + ci::vec2(0,emHeight), opts);
guiFont->drawString(text, ci::vec2((int)textSize.x*-0.5, 0) + pos + ci::vec2(0,emHeight), opts);
guiFont->drawString(text, pos + vec2(0,emHeight), opts);
ci::Font tmp("Arial", 16);
emHeight = floor( guiFont->measureString("M").y  );
`

to this

`
guiFont->drawString(text, ci::vec2(-textSize.x * 0.65, 0) + pos + ci::vec2(0,emHeight),gl::TextureFont::DrawOptions().scale( 0.5f ).pixelSnap( false ) );
guiFont->drawString(text, ci::vec2((int)textSize.x*-0.5, 0) + pos + ci::vec2(0,emHeight),gl::TextureFont::DrawOptions().scale( 0.5f ).pixelSnap( false ) );
guiFont->drawString(text, pos + vec2(0,emHeight), gl::TextureFont::DrawOptions().scale( 0.5f ).pixelSnap( false ) );
ci::Font tmp("Arial", 21);
emHeight = floor( guiFont->measureString("M").y * 0.55 );
`

Before and after screeshots's \/
https://imgur.com/gallery/oSiWjS2
